pkg/csi/plugins/nodeserver.go

```diff
@@ -577,3 +591,16 @@ func (ns *nodeServer) prepareSessMgr(workDir string) error {
 
 	return nil
 }
+
+func removeSymlink(targetPath string) (*csi.NodeUnpublishVolumeResponse, error) {
+	glog.Infof("remove symlink targetPath %v", targetPath)
```

Comment:
I think this logic should determine if the path is symlink , then do the remove.

Reply:
done.